drivers/ieee802154/ieee802154_kw41z.c

```diff
@@ -670,6 +998,17 @@ static void kw41z_iface_init(struct net_if *iface)
 	struct kw41z_context *kw41z = dev->driver_data;
 	u8_t *mac = get_mac(dev);
 
+#if KW41Z_DEBUG_MTB_ENABLED
+	MTB->POSITION = ((u32_t)(mtb - MTB->BASE)) & 0xFFFFFFF8;
+	MTB->FLOW = (((u32_t)mtb - MTB->BASE) +
+			(TRACE_BUFFER_SIZE * sizeof(u32_t))) & 0xFFFFFFF8;
+	MTB->MASTER = 0x80000000 + (TRACE_BUFFER_MAGNITUDE_PACKETS - 1);
+#endif
```

Comment:
as @galak noticed it already, try to create macros for all the magic numbers. It's too cryptic otherwise.